docs/community/release.md

```diff
@@ -191,13 +191,15 @@ The tag pattern is `v${RELEASE_VERSION}-rc${RELEASE_RC_NO}`, e.g. `v1.7.0-rc0`
 ```shell
 # Bump to the release version
 build/mvn versions:set -DgenerateBackupPoms=false -DnewVersion="${RELEASE_VERSION}"
+(cd kyuubi-server/web-ui && npm version "${RELEASE_VERSION}")
```

Comment:
need to run the command `npm install` to update the file `package-lock.json`?

https://github.com/apache/kyuubi/blob/55feb2b58c05912cc5158c8677ed48b2a0f7bf68/kyuubi-server/web-ui/package-lock.json#L2-L21

Reply:
`npm version` handles that